buildSrc/src/main/java/com/uber/okbuck/core/task/IntegrationTestTask.java

```diff
@@ -0,0 +1,7 @@
+package com.uber.okbuck.core.task;
+
+import org.gradle.api.tasks.testing.Test;
+
+public class IntegrationTestTask extends Test {
```

Comment:
Can you add some documentation on why this class exists and is empty?

Reply:
Nice catch, this should be deleted